.github/workflows/cd.yml

```diff
@@ -42,15 +42,15 @@ jobs:
         run: echo "::set-output name=release::${{ steps.semrel.outputs.version }}"
 
   publish:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
i'm not sure how long this is going to fly for; we may need to figure out how to get this running on ubuntu-20.04 or dig deeper into the networking issues that are causing the failures on 22.04, but for now I think we can just keep an eye on the bill.